libcudacxx/docs/extended_api/synchronization_primitives/barrier.md

```diff
@@ -60,6 +61,43 @@ Programs shall ensure that this transformation would not introduce errors, for
 Under CUDA Compute Capability 6 (Pascal) or prior, an object of type
   `cuda::barrier` or `cuda::std::barrier` may not be used.
 
+## Shared memory barriers with transaction count
+
+Starting with the Hopper architecture (CUDA Compute Capability 9), a
+`cuda::barrier` object located in shared memory supports a new count, called
+[tx-count](https://docs.nvidia.com/cuda/parallel-thread-execution/index.html#tracking-asynchronous-operations-by-the-mbarrier-object),
+which is used for tracking the completion of asynchronous memory operations or
```

Comment:
```suggestion
which is used for tracking the completion of some asynchronous memory operations or
```

Reply:
In fact, none of the currently exposed asynchronous memory operations after this PR uses the tx count.